src/Symfony/Component/WebLink/Link.php

```diff
@@ -46,7 +46,7 @@ class Link implements EvolvableLinkInterface
     private $rel = [];
 
     /**
-     * @var string[]
+     * @var array<string, string|bool|string[]>
```

Comment:
Note: On 6.0, this change collided with #41507. I've picked the version from this PR, to make the type consistent across the branches.